neural_cf.ipynb

```diff
@@ -10,57 +10,39 @@
   },
```

Comment:
We should probably make a new directory for keeping the notebooks

Reply:
If notebooks are not in project root directory, there will be problems with imports inside them. On the beginning I've created special directory for notebooks but it was very problematic, so I switched to to root dir.

Reply:
Same goes for tenorboard_script.py btw